synchronization/src/androidTestMovebis/java/de/cyface/synchronization/SyncAdapterTest.java

```diff
@@ -58,7 +58,7 @@ public void testRequestSync() throws InterruptedException {
         Account newAccount = new Account(TestUtils.DEFAULT_USERNAME, ACCOUNT_TYPE);
         if (am.addAccountExplicitly(newAccount, TestUtils.DEFAULT_PASSWORD, Bundle.EMPTY)) {
             ContentResolver.setIsSyncable(newAccount, AUTHORITY, 1);
-            ContentResolver.setSyncAutomatically(newAccount, AUTHORITY, true);
+            ContentResolver.addPeriodicSync(newAccount, AUTHORITY, Bundle.EMPTY, 1);
```

Comment:
The combination of syncAutomatically + periodic sync did not work as we expected which lead to the sync starting when though there is no wifi network. We now only use syncPeriodically and when it's activated (e.g. just connected to wifi) we trigger a initial sync which is what the user expects.